package.json

```diff
@@ -18,9 +18,13 @@
   "repository": "https://github.com/openzipkin/zipkin-js",
   "devDependencies": {
     "babel-cli": "^6.23.0",
+    "babel-polyfill": "^6.26.0",
     "babel-preset-es2015": "^6.18.0",
     "babel-register": "^6.23.0",
+    "buster": "^0.7.18",
     "chai": "^3.5.0",
+    "co": "^4.6.0",
+    "co-mocha": "^1.2.1",
```

Comment:
@mkante could we double check that we are adding minimal dependencies here? I am also curious about `buster`.

Reply:
Ok let me try

Reply:
I tried to replace `co and co-mocha` with `async/await` but the linter wasn't happy so I move then to `devDependencies`